modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/contestable_issues_controller.rb

```diff
@@ -169,11 +168,16 @@ def validate_json_schema
         validate_params
       end
 
-      def validate_json_schema_for_headers
-        AppealsApi::FormSchemas.new(
+      def form_schemas
+        @form_schemas ||= AppealsApi::FormSchemas.new(
           SCHEMA_ERROR_TYPE,
+          api_name: 'decision_reviews',
           schema_version: 'v2'
-        ).validate!(self.class::FORM_NUMBER, request_headers)
+        )
+      end
+
+      def validate_json_schema_for_headers
+        form_schemas.validate!('CONTESTABLE_ISSUES_HEADERS', request_headers)
```

Comment:
Previously, we used the `FORM_NUMBER` to locate the headers schema for each API - now that `FORM_NUMBER` only refers to numbers of forms, I needed to update the `validate_json_schema_for_headers` method in a few controllers to select the right schema - these `validate_json_schema_for_headers` methods are the only places we need to remember/specify these schema names, though.